API-Server/src/main/java/swm/hkcc/LGTM/app/modules/mission/service/HomeServiceImpl.java

```diff
@@ -43,7 +45,8 @@ public ServerDrivenScreenResponse getHomeScreen(Long memberId, int version) {
 
     private void processMissionContentType(Long memberId, MissionContentType missionContentType, List<ServerDrivenContent> serverDrivenContentList) {
         if (missionContentType.getViewType() == ViewType.ITEM) {
-            Function<Long, MissionContentData> missionListFunction = missionItemHolder.getMissionListFunction(missionContentType);
+            String memberType = memberService.getMemberType(memberId);
```

Comment:
memberType을 enum으로 지정하면 어떨까요?

```suggestion
            MemberType memberType = memberService.getMemberType(memberId);
```

Reply:
생각보다 해당 멤버의 타입을 많이 사용햐는 것 같아서 Member 엔티티 필드에 추가해야 된다고 생각했습니다. 그런데 추가해서 리펙토링하기에는 수정 사항이 많아져서  일단 스트링으로 사용했습니다. 
엔티티에 추가하도록 바꾸는건 어떻까요?